setup32.cfg

```diff
@@ -0,0 +1,18 @@
+[aliases]
+# python2.7 has upgraded unittest and it is no longer compatible with some
+# of our tests, so we run all through nose
+test = nosetests
+
+[nosetests]
+# nosetests skips test files with the executable bit by default
+# which can silently hide failing tests.
+# There are no executable scripts within the scikit-learn project
+# so let's turn the --exe flag on to avoid skipping tests by
+# mistake.
+exe = 1
+cover-html = 1
+cover-html-dir = coverage
+cover-package = sklearn
+
+detailed-errors = 1
+with-doctest = 0
```

Comment:
(I also totally agree that the solution was ugly, but the best I found. I really really don't like logic in Make files)